package.json

```diff
@@ -133,24 +128,17 @@
     "@types/react-dom": "~17.0.10",
     "@types/react-notification-system": "~0.2.41",
     "@types/react-redux": "~7.1.16",
-    "@types/react-router": "~5.1.20",
     "@types/react-router-dom": "~5.3.0",
     "@typescript-eslint/eslint-plugin": "~5.10.0",
     "@typescript-eslint/parser": "~5.10.0",
-    "acorn": "~8.7.0",
-    "assert": "~2.0.0",
-    "babel-core": "~7.0.0-bridge.0",
     "babel-loader": "~8.2.3",
     "babel-plugin-formatjs": "~10.3.12",
-    "babel-polyfill": "~6.26.0",
-    "buffer": "~6.0.3",
     "chai": "~3.5.0",
     "chai-as-promised": "~7.1.1",
     "chai-http": "~4.3.0",
     "css-loader": "~6.4.0",
     "eslint": "~8.41.0",
     "eslint-plugin-jsdoc": "~39.2.9",
-    "eslint-plugin-react": "~7.32.2",
```

Comment:
While this isn't used in the current eslint configuration, `eslint-plugin-react` might be relevant. After doing some digging in the file history, it looks like the packages was added but the plugin was never added to eslintric.

I tested adding the package myself, and after running an eslint check with it enabled, there were no issues. Seeins as it looks like OED has never used the package but has no issues, it can probably just be removed.

Reply:
@spearec: I'll leave it off this PR but if you want to create a separate PR to enable it then that is fine with me.